### DISCUSSION TITLE
Issue with Displaying Own MMSI on OpenCPN

### DISCUSSION THREAD

**Discussion by 79baracuda79** [jvde-github/AIS-catcher#327] [Q&A] [answered]
Hi,

I'm trying to configure AIS-Catcher to display my own ship's MMSI on OpenCPN. I’ve tried using both the -N and -O parameters to set my ship's MMSI, but so far, I haven't been able to see it properly on OpenCPN.

Could you clarify if there’s an additional step or configuration needed? Is there something I’m missing in the setup?
Thank you for your assistance!

**Comment by jvde-github** [comment]
If you start AIS-catcher with -O then it uses AIVDO instead of AIVDM for the NMEA line:
```
$ AIS-catcher -O 212396000
AIS-catcher (build Nov 18 2024) v0.60-312-g4f7b402d
(C) Copyright 2021-2024 jvde-github and other contributors
This is free software; see the source for copying conditions.There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Found Rafael Micro R820T tuner
!AIVDO,1,1,,B,13:SWp30000Bq@VMfG=Kos>B0<BV,0*1F ( MSG: 1, REPEAT: 0, MMSI: 212396000, signalpower: -15.7594, ppm: -2.31481, timestamp: 20241118201136)
```
To be honest, don't know how opencpn deals with this. Will have a look.

**Comment by 79baracuda79** [comment]


**Reply by jvde-github** [reply]
That is an interesting observation. Do you see a different in the AIS stream between AISrec and AIS-catcher with -O? How do you send it to opencpn?

**Comment by 79baracuda79** [comment]


**Reply by jvde-github** [reply, marked as answer]
It sounds like you do not have the latest version. Can you install Edge from [here](https://github.com/jvde-github/AIS-catcher#windows-binaries)?
